cmd/ferretdb/main.go

```diff
@@ -124,6 +124,13 @@ var hanaFlags struct {
 	HANAURL string `name:"hana-url" help:"SAP HANA URL for 'hana' handler"`
 }
 
+// The mySQLFlags struct represents flags that are used by the "mysql" backend.
+//
+// See main_mysql.go.
+var mySQLFlags struct {
+	MySQLURL string `name:"mysql-url" default:"127.0.0.1:3306/ferretdb" help:"MySQL URL for 'mysql' handler"`
```

Comment:
Let's add `hidden:""` for now.

Also, let's use `mysql://` scheme for consistency with other backends